### PR TITLE
fix issue with Chrome and whitespaces (#2365)

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Comment by AlexKamaev** [comment]
\cc @AndreyBelym

File: src/client/automation/playback/type/type-text.js [hunk 1]
```diff
@@ -145,6 +145,9 @@ function _typeTextToContentEditable (element, text) {
     var endNode             = currentSelection.endPos.node;
     var needProcessInput    = true;
     var needRaiseInputEvent = true;
+    var textInputData       = text;
+
+    text = text === ' ' ? String.fromCharCode(160) : text;
```
Comment (AndreyBelym) [inline src/client/automation/playback/type/type-text.js]: Text won't be a single char when `typingOptions.paste` is set to `true`. https://devexpress.github.io/testcafe/documentation/test-api/actions/action-options.html#typing-action-options

File: test/client/fixtures/automation/content-editable/regression-test/index-test.js [hunk 2]
```diff
@@ -107,7 +106,39 @@ $(document).ready(function () {
...
+                $body.attr('contenteditable', 'false');
+
                 startNext();
             });
     });
+
+    if (!browserUtils.isFirefox) {
+        asyncTest('textInput eventArgs.data should contain space but not &nbsp;)', function () {
+            var editor = document.createElement('div');
+            var text   = ' ';
```
Comment (AndreyBelym) [inline test/client/fixtures/automation/content-editable/regression-test/index-test.js]: Use more complex complex value here (at least `Hello   world`).

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

File: src/client/automation/playback/type/type-text.js [hunk 3]
```diff
@@ -11,6 +11,7 @@ var domUtils        = testCafeCore.domUtils;
 var contentEditable = testCafeCore.contentEditable;
 var textSelection   = testCafeCore.textSelection;
 
+const WHITE_SPACES_RE = /\s/g;
```
Comment (AndreyBelym) [inline src/client/automation/playback/type/type-text.js]: Don't use `\s`, because it matches newline characters.

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Pull request merged**